src/auth/action-code-settings-builder.ts

```diff
@@ -94,15 +94,27 @@ export interface ActionCodeSettings {
    * configured per project. This field provides the ability to explicitly choose
    * configured per project. This fields provides the ability explicitly choose
    * one. If none is provided, the oldest domain is used by default.
+   * @deprecated use linkDomain instead
```

Comment:
Is there a way to link to the new property? `{@link ActionCodeSettings#linkDomain }`, if that doesn't work, could we use backtick here?

Reply:
Done.  Used backtick instead.